DiceKitTests/Die_Tests.swift

```diff
@@ -268,55 +247,50 @@ extension Die_Tests {
     
     func common_RollerType_shouldReturnWithinPositiveRangeForSidesGreaterThan1(rollerType: Die.RollerType) {
         property("generates values within range of 1...sides") <- forAll {
-            (i: PositiveSidesType) in
-            
-            guard i > 1 else { return true }
-            
-            let sides = Int(i)
-            
-            let result = rollerType(sides: sides)
+            (i: Int) in
             
-            return result > 0 && result <= sides
+            return (i > 1) ==> {
+                let result = rollerType(sides: i)
+                
+                return result > 0 && result <= i
+            }
         }
     }
     
     func common_RollerType_shouldReturnWithinNegativeRangeForSidesLessThan0(rollerType: Die.RollerType) {
         property("generates values within range of sides..<0") <- forAll {
-            (i: PositiveSidesType) in
-            
-            guard i > 1 else { return true }
-            
-            let sides = -Int(i)
-            
-            let result = rollerType(sides: sides)
+            (i: Int) in
             
-            return result < 0 && result >= sides
+            return (i < -1) ==> {
+                let result = rollerType(sides: i)
+                
+                return result < 0 && result >= i
+            }
         }
     }
     
     func common_RollerType_shouldReturnWithinRangeForSides(rollerType: Die.RollerType) {
         property("generates values within range of -sides...sides") <- forAll {
-            (i: PositiveSidesType) in
-            
-            let sides = Int(i)
-            
-            let result = rollerType(sides: sides)
+            (i: Int) in
             
-            return (-sides...sides).contains(result)
+            // Int.min has one more value than Int.max, making this crash when negating it
```

Comment:
This comment doesn't make much sense anymore. Before it used to simply prevent `Int.min`. If you negate that you get too large of a number. But, if you have a negative number at all, then the range is reversed. I think I can actually do the same check, but then do `abs` to ensure that the range is correct.